src/js/relayserver/runServer.js

```diff
@@ -15,6 +15,7 @@ const argv = parseArgs(process.argv.slice(2), {
       'PercentFee',
       'Url',
       'RelayHubAddress',
+      'StakeManagerAddress',
```

Comment:
no need to pass SM as parameter. it has to be the `relayHub.stakeManager`.

Reply:
Correct. Will fix. (Next PR)